source/includes/byok-encryption-recommendations-short.rst

```diff
@@ -0,0 +1,5 @@
+For staging and production environments, we
+recommend enabling BYOK encryption when provisioning your {+clusters+}.
+For development and testing, consider skipping BYOK encryption 
```

Comment:
```suggestion
For development and testing, consider skipping |byok| encryption 
```
Question/Suggestion: Should each instance of BYOK not in a link be the substitution? If so, applies to all instances.